### PR TITLE
Add fun fact command

### PR DESCRIPTION
**Pull request by Atmois**
Unfinished as going to add more fun facts.

## Summary by Sourcery

Add a new command to the bot that delivers random fun facts about Linux, enhancing user engagement with interesting trivia.

New Features:
- Introduce a new 'fun fact' command in the bot that provides users with random Linux-related fun facts.

### CODE REVIEW

**Comment by HikariNee** [comment]
You can add this too.
https://news.ycombinator.com/item?id=8641073

**Comment by electron271** [comment]
@Atmois is it fine if i can unmark as draft or should i wait until you add more facts

File: tux/cogs/fun/fact.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            "96.3% of the top 1,000,000 web servers were reported to run on Linux.",
+            "In the early 2000s, Steve Jobs, who at the time was the CEO of Apple, offered a job to Linus Torvalds to work on OSX which Torvalds declined.",
+            "Linus Torvalds said that he would have never created Linux if FreeBSD had been available at the time.",
+            "Linux is used by every major space programme in the world including NASA, SpaceX, and the European Space Agency.",
+        ]
+
+    @commands.hybrid_command(
+        name="fact",
+        aliases=["funfact"],
+        usage="fact",
```
[the remark this answers is not in the document]
Comment (electron271) [reply tux/cogs/fun/fact.py]: fixed

**Comment by Atmois** [comment]
Go ahead

**Review by electron271 — APPROVED** [review]

**Pull request merged**